upgrade-monitoring-services.md

```diff
@@ -0,0 +1,111 @@
+
+In the following upgrade steps, you need to download the Grafana installation package of your desired version from the Grafana website, and then use it to create a Grafana package that TiUP can use.
+
+### Step 1. Download a new Grafana installation package from the Grafana website
+
+Download a new installation package from the [Grafana download page](https://grafana.com/grafana/download?pg=get&plcmt=selfmanaged-box1-cta1) and extract it.
+
+### Step 2. Download the Grafana installation package provided by TiDB
+
+1. Download the `TiDB-community-server` package from the [TiDB download page](https://pingcap.com/product/#SelectProduct) and extract it.
+2. In the extracted files, locate `grafana-v{version}-linux-amd64.tar.gz` and extract it.
+
+    ```bash
+    tar -xzf grafana-v{version}-linux-amd64.tar.gz
+    ```
+
+### Step 3. Create a new Grafana package that TiUP can use
+
+1. Copy the files extracted in [Step 1](#step-1-download-a-new-grafana-installation-package-from-the-grafana-website), and then use the copied files to replace the files in the `./grafana-v{version}-linux-amd64/` directory extracted in [Step 2](#step-2-download-the-grafana-installation-package-provided-by-tidb).
+2. Recompress the `./grafana-v{version}-linux-amd64` directory and name the new compressed package as `grafana-v{new-version}.tar.gz`, where `{new-version}` can be specified according to your need.
+
+    ```bash
+    cd grafana-v{version}-linux-amd64.tar.gz
+    tar -zcvf ../grafana-v{new-version}.tar.gz ./
+    ```
+
+### Step 4. Upgrade Grafana using the newly created Grafana package
+
+Execute the following command to upgrade Grafana:
+
+```bash
+tiup cluster patch <cluster-name> grafana-v{new-version}.tar.gz -R grafana
+
+```
+
+After the upgrade, you can go to the home page of the Grafana server (usually at `http://<Grafana-server-host-name>:3000`), and then check the Grafana version on the page to confirm whether the upgrade is successful.
+
+## Upgrade AlertManager
+
+The AlertManager package in the TiDB installation package is directly from the Prometheus website. Therefore, when upgrading AlertManager, you only need to download and install a new version of AlertManager from the Prometheus website.
+
+### Step 1. Download a new AlertManager installation package from the Prometheus website
+
+Download the `alertmanager` installation package from the [Prometheus download page](https://prometheus.io/download/#alertmanager).
+
+### Step 2. Upgrade AlertManager using the downloaded installation package
+
+Execute the following command to upgrade AlertManager:
+
+```bash
+tiup cluster patch <cluster-name> alertmanager-v{new-version}-linux-amd64.tar.gz -R alertmanager
+```
```

Comment:
```suggestion
```

After the upgrade, you can go to the home page of the AlertManager server (usually at `http://<Alertmanager-server-host-name>:9093`), click **Status** in the top navigation menu, and then check the AlertManager version and confirm whether the upgrade is successful.
```